src/serialize_sequence.cc

```diff
@@ -47,10 +49,9 @@ void remove_path(const std::filesystem::path& folder)
     }
 }
 
-/// Extracted from High Level Controls Utility Library (DESY), file string_util.h/cc
 std::string escape_filename_characters(gul14::string_view str)
 {
-    static const std::string bad_characters("/\\:?*\"\'<>|$&");
+    static const std::string bad_characters("/\\:?*\"\'<>|$&[]");
```

Comment:
HA!

I knew it.
Opposing this change, of course. Instead fix the code ;-q

Reply:
Reverted.